### DISCUSSION TITLE
How do I manage Roles & Permissions through configuration

### DISCUSSION THREAD

**Discussion by periabyte** [strapi/strapi#6319] [Legacy Topics]
**Background:**
We're starting this project where we will have multiple developers working on strapi, Ideally everyone should have the same settings as everybody else with out running migrations everytime they update their local copy or manually updating through the Dashboard.

Is it possible with the current release? or does anyone have an experience implementing something like this?

**Comment by derrickmehaffy** [comment]
Another discussion that might help here: https://github.com/strapi/strapi/discussions/6294
